.github/workflows/node.yml

```diff
@@ -0,0 +1,35 @@
+name: Node.js CI
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  builds:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [10.x, 12.x, 14.x]
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+    
+    - name: Install dependencies
+      working-directory: ./ui
```

Comment:
Why `working-directory` on each build over using a `default`?

Reply:
yeah so I know originally I was referencing [this](https://dev.to/shofol/run-your-github-actions-jobs-from-a-specific-directory-1i9e) but it wasn't getting recognized for some reason in github actions so I split it up for [individual jobs](https://github.community/t/use-working-directory-for-entire-job/16747/6)